ci/templatetags/seed.py

```diff
@@ -8,3 +8,15 @@
 @register.simple_tag
 def current_version():
     return seed_control_interface.__version__
+
+
+@register.filter
+def cleanup(value):
+    return value.replace('_', ' ')
+
+
+@register.filter
+def check_object(value):
```

Comment:
`is_dict` is possibly a better name for this function. Makes it easier to see what the filter is doing in the template.